compose/normandy-base.yml

```diff
@@ -14,4 +14,6 @@ services:
       DJANGO_CAN_EDIT_ACTIONS_IN_USE: "true"
       # Pulled from the local environment
       DJANGO_API_CACHE_ENABLED:
+      DJANGO_DEBUG: 1
+      DEBUG: 1
```

Comment:
@mythmon Should the compose setup be set to debug mode?

In any case, `DJANGO_DEBUG` should be the only necessary environment variable here.